packages/performant-ui/src/components/Pagination.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useMemo } from 'react';
+import Button from './Button';
+import { HiArrowLongLeft, HiArrowLongRight, HiChevronLeft, HiChevronRight } from 'react-icons/hi2';
+import { FaChevronLeft, FaChevronRight } from 'react-icons/fa';
+
+interface Props {
+  className?: string
+  current: number
+  onChange: (page: number) => void
+  pageCount: number
+}
+
+const getPageBatches = (count: number, current: number) => {
+  if (count <= 7) {
```

Comment:
Could we extract the "magic numbers" in this function to constants to provide a little more context?